.github/workflows/build.yml

```diff
@@ -9,9 +9,6 @@ on:
       - main
     paths-ignore:
       - "**.md"
-  schedule:
```

Comment:
This file shouldn't be edited

Reply:
Now the file is not included in the pull request anymore.

Reply:
I randomly have the following error in (master,get). One day it occurs and the other day not. But I see the same kind of error in the build of the original branch so I guess this is not related to this pull request content.

 _[custom_lint]: ::group::❌ test/server_test.dart: hot-reload handles the source change of one plugin and restart it (failed)
[custom_lint]: **PathNotFoundException**: Failed to watch path, path = '/home/runner/work/dart_custom_lint/dart_custom_lint/packages/custom_lint/.dart_tool/**custom_lint_testBZZXHU' (OS Error: No such file or directory**, errno = 2)_